config/application.rb

```diff
@@ -1,7 +1,9 @@
 require_relative "boot"
 
+require "rails"
 # Pick the frameworks you want:
 require "active_model/railtie"
+require "active_job/railtie"
```

Comment:
would be better to add this commented-out, as the application doesn't use ActiveJob